internal/cmd/commands/authmethodscmd/oidc_funcs.go

```diff
@@ -33,6 +33,7 @@ type extraOidcCmdVars struct {
 	flagAccountClaimMaps                  []string
 	flagDisableDiscoveredConfigValidation bool
 	flagDryRun                            bool
+	flagPrompts                           []string
```

Comment:
Just checking: did you test the CLI by hand and verify that it works as expected
with the new prompts?

Reply:
Yes I did. I'll do it and add the output to the PR